### DISCUSSION TITLE
[Feature] Use multiple tasks inside "scripts" package.json

### DISCUSSION THREAD

**Discussion by ColtHands** [npm/rfcs#216] [RRFC]
I think it would be a good idea to optionally chain multiple scripts as an array of strings and run them asynchronously.

```json
"scripts": {
  "build": [ "webpack --verbose", "npm run test", "git add .", "echo 'all done'" ]
}
```

**Comment by ljharb** [comment]
Duplicate/alternate of https://github.com/npm/rfcs/issues/190
